### PR TITLE
chore(release): Version bump and changelog for release

### PR DESCRIPTION
**Pull request by mgiambalvo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "@types/node": "^6.0.46",
     "@types/q": "^0.0.32",
     "@types/selenium-webdriver": "~2.53.39",
-    "blocking-proxy": "0.0.5",
+    "blocking-proxy": "^0.0.5",
```
Comment (juliemr) [inline package.json]: I'm a bit worried about allowing `^` since this is pre version 1 and versioning is nebulous -can we leave it precisely pinned?

Comment (mgiambalvo) [inline package.json]: Yeah, I was hoping to be able to get out fixes without having to publish a new Protractor. But you're right, with things as they are it's probably better to just leave pinned.

**Comment by wswebcreation** [comment]
@mgiambalvo,

Maybe a stupid question, but protractor uses semver and the commits are features, shouldn't protractor be updated to 5.2.0 in stead of a fix release?

Tnx in advance

**Comment by mgiambalvo** [comment]
The main purpose of this release was to undo our accidental dependency on TS 2.1 (which was an unintentional breaking change) and fix support for TS 2.0. There were a couple tiny, low risk features that also made it in, so I left it at 5.1.1. Maybe you're right though, it could have easily been 5.2.0

**Comment by wswebcreation** [comment]
Tnx for explaining, it's clear to me.

**Pull request merged**